pkgs/os-specific/linux/kernel/linux-6.5.nix

```diff
@@ -3,7 +3,7 @@
 with lib;
 
 buildLinux (args // rec {
-  version = "6.5.1";
+  version = "6.5.2";
```

Comment:
Version bumped, but not hash; Please, do not merge until this is fixed.

Reply:
Done.